src/Documentation/glossary.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Glossary",
+  "desc": "This guide is aimed to familiarize the users with definitions to relevant DVC concepts and terminologies which are frequently used.",
```

Comment:
Is is posisble to wrap JSON files to 80 chars? Otherwise a JS file may be preferable after all.

Reply:
agreed w @jorgeorpinel - we should run it by prettier